doc_source/aws-properties-ec2-security-group-ingress.md

```diff
@@ -351,3 +351,57 @@ SGPing:
       ToPort: -1
       CidrIp: 10.0.0.0/24
 ```
+
+### Allow Access from a Security Group within a Peered VPC
+
+In some cases, it may become necessary to allow specific traffic from a targeted originating (source) security group in a different AWS Account to specific ports on the target. An example of this scenario could include a one-way security scanning resource that exists in a separate AWS Account that needs to run diagnostics into a peered account. The following snippet creates a security group that allows this access on Port 80 from the client side, ensuring only desired resources are scanned.
+
+#### JSON<a name="aws-resource-ec2-securitygroupingress-example-4.json"></a>
+
+```
+{
+  "AWSTemplateFormatVersion": "2010-09-09",
+  "Resources": {
+    "SecurityScanGroup": {
+      "Type": "AWS::EC2::SecurityGroup",
+      "Properties": {
+        "VpcId": "vpc-e063f789",
+        "GroupDescription": "Security Group allowing ingress for Security Scanners"
+      }
+    },
+    "SecurityScanGroupIngress": {
+      "Type": "AWS::EC2::SecurityGroupIngress",
+      "Properties": {
+        "GroupId": "SecurityScanGroup.GroupId",
```

Comment:
Perhaps this could be improved with a `Ref` to the `SecurityScanGroup` in the template. That would also establish deployment ordering.

Reply:
I believe that the !GetAtt also establishes ordering - and since the SecurityScanGroup exists within a VPC, I belive that !Ref will return the SecurityGroup Name rather than the Group ID. I'm going off how the [documentation reads](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ec2-security-group.html#w2ab1c21c10d473c15) here.